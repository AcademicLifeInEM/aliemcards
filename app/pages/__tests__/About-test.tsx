jest.mock('react-router-dom');

import * as React from 'react';
import * as renderer from 'react-test-renderer';
import About from '../About';

describe('<About />', () => {
  it('should render correctly', () => {
    const component = renderer.create(<About />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
