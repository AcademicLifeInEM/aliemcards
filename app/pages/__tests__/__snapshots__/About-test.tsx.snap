// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> should render correctly 1`] = `
<div
  className="content"
>
  <h1>
    About
  </h1>
  <h2>
    Paucis Verbis Cards
  </h2>
  <p>
    The Paucis Verbis (PV) cards were initially developed as a bedside teaching tool used by Academic Life in Emergency Medicine (ALiEM) Founder and Editor-in-chief Michelle Lin. They were short, succinct, rapid-fire summaries of key learning points on various Emergency Medicine (EM) topics that would fit on a 5 x 7 index card. Michelle carried these cards in her pocket and handed them out to trainees as needed. Eventually others at her institution began using the cards and as digital and mobile technology became more ubiquitous, Michelle transitioned the cards to several mobile platforms so that she could share them with the greater EM community.
  </p>
  <h2>
    Now ALiEM Cards
  </h2>
  <p>
    As the
     
    <a
      href="http://www.aliem.com"
      rel="noopener noreferrer"
      target="_blank"
    >
      ALiEM
    </a>
     
    blog has grown, so have the PV Cards. What began as a handful of personal notes, are now a key FOAMed point-of-care resource.
  </p>
  <p>
    This website is the next iteration of that project.
  </p>
  <h2>
    Help us improve this resource
  </h2>
  <p>
    Please 
     how you use these cards. What would make them more useful? Found an error? Help us fix it!
  </p>
  <h2>
    The ALiEM Cards Team
  </h2>
  <div
    className="row row--wrap"
  >
    <div
      className="column column--relative-50"
    >
      <h3>
        Michelle Lin, MD
      </h3>
      <div>
        <div>
          <strong>
            Creator & Editor-in-Chief
          </strong>
        </div>
        <div>
          Endowed Chair for EM Education
        </div>
        <div>
          UC San Francisco / Zuckerberg San Francisco General
        </div>
        <div>
          <a
            href="https://www.twitter.com/M_Lin"
          >
            @
            M_Lin
          </a>
        </div>
      </div>
    </div>
    <div
      className="column column--relative-50"
    >
      <h3>
        Jeremy Voros, MD
      </h3>
      <div>
        <div>
          <strong>
            Developer
          </strong>
        </div>
        <div>
          Attending Physician
        </div>
        <div>
          Emergency Physicians Integrated Care, Utah
        </div>
        <div>
          <a
            href="https://www.twitter.com/VorosMD"
          >
            @
            VorosMD
          </a>
        </div>
      </div>
    </div>
    <div
      className="column column--relative-50"
    >
      <h3>
        Derek Sifford, FP-C
      </h3>
      <div>
        <div>
          <strong>
            Developer
          </strong>
        </div>
        <div>
          Paramedic
        </div>
        <div>
          Wayne State University
        </div>
        <div>
          <a
            href="https://www.twitter.com/flightmed1"
          >
            @
            flightmed1
          </a>
        </div>
      </div>
    </div>
    <div
      className="column column--relative-50"
    >
      <h3>
        Catherine Patocka, MD, MHPE
      </h3>
      <div>
        <div>
          <strong>
            Research Lead
          </strong>
        </div>
        <div>
          Assistant Professor
        </div>
        <div>
          University of Calgary
        </div>
        <div>
          <a
            href="https://www.twitter.com/patockaem"
          >
            @
            patockaem
          </a>
        </div>
      </div>
    </div>
    <div
      className="column column--relative-50"
    >
      <h3>
        Teresa Chan, MD MHPE
      </h3>
      <div>
        <div>
          <strong>
            Research Mentor
          </strong>
        </div>
        <div>
          Emergency Physician and Base Hospital Physician
        </div>
        <div>
          Hamilton Health Sciences
        </div>
        <div>
          <a
            href="https://www.twitter.com/TChanMD"
          >
            @
            TChanMD
          </a>
        </div>
      </div>
    </div>
    <div
      className="column column--relative-50"
    >
      <h3>
        Andy Little, DO
      </h3>
      <div>
        <div>
          <strong>
            Content Editor
          </strong>
        </div>
        <div>
          Attending Physician, Core Faculty
        </div>
        <div>
          Doctors Hospital Emergency Medicine Residency
        </div>
        <div>
          <a
            href="https://www.twitter.com/andyglittle"
          >
            @
            andyglittle
          </a>
        </div>
      </div>
    </div>
    <div
      className="column column--relative-50"
    >
      <h3>
        Layla Abubshait, MD
      </h3>
      <div>
        <div>
          <strong>
            Faculty Development and Medical Education Fellow
          </strong>
        </div>
        <div>
          Department of Emergency Medicine
        </div>
        <div>
          Beth Israel Deaconess Medical Center
        </div>
        <div>
          <a
            href="https://www.twitter.com/layla_abubshait"
          >
            @
            layla_abubshait
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
